### PR TITLE
Add an explanation for auto saving based on specific languages/file types

### PR DESCRIPTION
**Pull request by cachandlerdev**
I added a short explanation of how to customize the auto save feature for specific language types in accordance with [this issue](https://github.com/microsoft/vscode/issues/234769), since the feature is currently not mentioned in the documentation or in the Settings GUI.

### CODE REVIEW

File: docs/editor/codebasics.md [hunk 1]
```diff
@@ -95,6 +95,13 @@ For more control over `Auto Save`, open User or Workspace [settings](/docs/getst
   * `onWindowChange` - to save files when the focus moves out of the VS Code window.
 * `setting(files.autoSaveDelay)`: Configures the delay in milliseconds when `setting(files.autoSave)` is configured to `afterDelay`. The default is 1000 ms.
 
+If you want to customize `Auto Save` features for specific languages or file types, you can do so from your `setting.json` by adding language specific rules. For example, you could disable `Auto Save` for LaTeX files like this:
```
Comment (ntrogh) [inline docs/editor/codebasics.md:98]:
```suggestion
If you want to customize the `Auto Save` functionality for specific languages or file types, you can do so from the `settings.json` file by adding language-specific rules. 

For example, to disable `Auto Save` for LaTeX files:

```

**Review by ntrogh — CHANGES_REQUESTED** [review]
@cachandlerdev thanks for updating the docs. Just added some text suggestions.

**Comment by cachandlerdev** [comment]
I have made the changes as requested.

**Review by ntrogh — APPROVED** [review]
@cachandlerdev thanks for contributing to our docs!

**Review by lszomoru — APPROVED** [review]

**Pull request merged**